build-tools/xaprepare/xaprepare/ConfigAndData/Dependencies/Linux.DebianCommon.cs

```diff
@@ -13,6 +13,7 @@ abstract class LinuxDebianCommon : Linux
 			new DebianLinuxProgram ("cli-common-dev"),
 			new DebianLinuxProgram ("curl"),
 			new DebianLinuxProgram ("devscripts"),
+			new DebianLinuxProgram ("dotnet-sdk-3.1"),
```

Comment:
There's one problem with this dependency - it's not in standard Debian/Ubuntu/etc repositories, so it will fail on an unprepared Debian machine. I think we should at least handle failure to detect or install it with a descriptive error message pointing to the dotnet website with instructions on how to add the repository.

Reply:
I've removed this, it looks like this .NET Core 3.1 installation issue is happening on other PRs as well, we can look to address this issue in a separate PR.